sdk/translation/azure-ai-translation-text/CHANGELOG.md

```diff
@@ -5,8 +5,10 @@
 ### Features Added
 
 ### Breaking Changes
+  - When using the Cognitive Services token, the audience is now required.
 
 ### Bugs Fixed
+  - Fixed a bug where Entra Id authentication couldn't be used with custom endpoint.
```

Comment:
I'm wondering what version this should be released with. It's breaking, but it never worked in the first place - right? @annatisch what are your thoughts?

Reply:
In what way would this be breaking exactly?

Reply:
I removed the breaking line in MD.